docs/docs/rest-api/public/api/traits.raml

```diff
@@ -0,0 +1,15 @@
+
+- secured:
+    responses:
+      401:
+        description: Authentication is enabled and you did not provide enough or wrong information to authenticate that request.
```

Comment:
I'd place * Unauthorized* in front, so one does not have to read the whole sentence ;)